jaeger-core/src/main/java/io/jaegertracing/internal/JaegerTracer.java

```diff
@@ -58,24 +58,24 @@
 import lombok.ToString;
 import lombok.extern.slf4j.Slf4j;
 
-@ToString(exclude = {"registry", "clock", "metrics", "scopeManager"})
+@ToString
 @Slf4j
 public class JaegerTracer implements Tracer, Closeable {
-
   private final String version;
   private final String serviceName;
   private final Reporter reporter;
   private final Sampler sampler;
-  private final PropagationRegistry registry;
-  private final Clock clock;
-  private final Metrics metrics;
-  private final int ipv4;
   private final Map<String, ?> tags;
   private final boolean zipkinSharedRpcSpan;
-  private final ScopeManager scopeManager;
-  private final BaggageSetter baggageSetter;
   private final boolean expandExceptionLogs;
-  private final JaegerObjectFactory objectFactory;
+
+  @ToString.Exclude private final PropagationRegistry registry;
```

Comment:
I would like more adding `@ToString.Exclude` directly on top of the excluded field without creating a new block for excluded fields. And keep the field at the original place

```java
private final boolean zipkinSharedRpcSpan
@ToString.Exclude
private final ScopeManager scopeManager;
```